### PR TITLE
ReEdit some code

### PR DESCRIPTION
**Pull request by TolkachDima**
Change Success Checkout Link;
Built generalized clickOnCheckoutLink method;
Edit in TC_DP194_37 @MethodSource

### CODE REVIEW

File: src/main/java/steps/CategoryPageStep.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public CategoryPageStep verifyCurrentPage(String expectedUrl) {
     }
 
 
-    public List<String> getItemNames(){
+    public List<String> getItemNames() {
```
Comment (ghost) [inline src/main/java/steps/CategoryPageStep.java]: Make it private or protected

File: src/main/java/steps/CategoryPageStep.java [hunk 2]
```diff
@@ -52,8 +52,8 @@ public CategoryPageStep selectShowValueNameLowestRatingStep() {
         return this;
     }
 
-    public CategoryPageStep compareResult(List<String> expected, List<String> actual){
-        assertEquals(expected,actual);
+    public CategoryPageStep compareResult(List<String> expected, List<String> actual) {
```
Comment (ghost) [inline src/main/java/steps/CategoryPageStep.java]: Remove "List<String> actual" parameter from method

File: src/main/java/steps/HomePageStep.java [hunk 3]
```diff
@@ -47,20 +47,22 @@ public CategoryPageStep clickPhoneNavBar(){
...
+//
+//    public CheckoutPageRegisterStep clickOnCheckoutLinkRegister(){
+//        homePage.getTopNavBar().clickCheckoutLink();
+//        return new CheckoutPageRegisterStep(this.getDriver());
+//    }
+//
+//    public CheckoutPageAsLoggedUserStep clickOnCheckoutLinkLogged(){
+//        homePage.getTopNavBar().clickCheckoutLink();
+//        return new CheckoutPageAsLoggedUserStep(this.getDriver());
+//    }
```
Comment (ghost) [inline src/main/java/steps/HomePageStep.java]: Remove or uncomment it

File: src/main/java/model/ProductItem.java [hunk 4]
```diff
@@ -17,11 +17,12 @@
     private TextBlock productPrice;
     @FindBy(className = "price-tax")
     private TextBlock productTax;
-    @FindBy(xpath = "div[2]/div[2]/button[1]/span")
+//    @FindBy(xpath = "div[@class = 'button-group']/button[1]")
```
Comment (light-di) [inline src/main/java/model/ProductItem.java]: delete it

**Review by light-di — APPROVED** [review]

**Review by Tbotny — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**